pkg/setting/setting_openfeature.go

```diff
@@ -0,0 +1,36 @@
+package setting
+
+const (
+	StaticProviderType = "static"
+	GOFFProviderType   = "goff"
+)
+
+type OpenFeatureSettings struct {
+	ProviderType string
+	URL          string
```

Comment:
maybe this could be a `*url.URL`, and we could parse it here (or leave `nil` if `ProviderType == "static"`)